assets/js/helpers.ts

```diff
@@ -104,4 +104,25 @@ function defaultZoneLabel(z: Zone): string {
   }
 }
 
-export { choosePage, alertsForLine, formatTime, defaultZoneLabel };
+function arincToRealtimeId(
+  stationZone: string,
+  line: string,
+  arincToRealtimeIdMap: { [key: string]: string },
```

Comment:
If `arincToRealtimeIdMap` is passed to the client as essentially a global constant (for the lifetime of the page) on the `window` object, does this function need to receive it as an argument? Does it need to be passed through the component tree as props?

If we don't expect it to ever change for the client's lifetime, I think it would be ok for the `return` line of this function to simply `return window.arincToRealtimeIdMap[stationZone];`.

Reply:
Ah yep you're totally right and I agree. This simplifies things a lot!